### DISCUSSION TITLE
any plans to support the latest AU version and imposter versions?

### DISCUSSION THREAD

**Discussion by GD825** [dimaguy/ImpostorHQ#32] [Ideas]
any plans to support the latest AU version and imposter versions?..

would like to use this on the latest versions of both if possible

**Comment by dimaguy** [comment]
Nope, there are no such plans. Unfortunately, among us has been on popularity decline and there is no incentive anymore to develop for the Impostor platform. Honestly I'm surprised I haven't archived this yet as the code doesn't even match the documentation anymore
